### ISSUE TITLE
Custom Validation error message to a particular field

### ISSUE THREAD

**Issue by dbpolito** [fuel/core#834]
Would be great if we could set a custom Validation error message to a particular field, for example:

``` php
$val = \Validation::forge($factory);
$val->add('field', 'Field Name')->add_rule('required')->set_message('Custom Error Message to this field')
```

**Comment by Savageman** [comment]
This feature is already available using `$val->set_message('field', 'Custom Error Message to this field');`

Are you requesting an alias in the Fieldset_Field class so you can chain it directly like in your example?

**Comment by dbpolito** [comment]
Actually when you use set_message you set for a rule and not for a field.

http://localhost/fuel/docs/classes/validation/methods.html#/method_set_message

**Comment by Savageman** [comment]
Doh. You're right, my bad. Sorry.

**Comment by jschreuder** [comment]
Closed in favor of #838

**Comment by jschreuder** [comment]
Implemented here: 899b67384f91e45a42368f593a226f8812a31984
